### ISSUE TITLE
React 15+ support

### ISSUE THREAD

**Issue by DanielRuf** [telerik/kendo-react-inputs#35]
@dmihaylo @KirilNN would be good to have React 15+ support.

**Comment by petyosi** [comment]
Hello @DanielRuf , 

Our react development has taken a backseat as we prioritized Angular 2. More details about that are available in the following blog post: http://www.telerik.com/blogs/kendo-ui-for-react-the-road-ahead. 

Of course, even though while this is not in active development right now, pull requests are more than welcome.

**Comment by DanielRuf** [comment]
@petyosi So no chance to get all the dependencies and the module ready for React 15 directly?

I am not 100% sure, but would it be sufficient to change the version number of the React dependencies to support React 15?

**Comment by DanielRuf** [comment]
Is this project still actively maintained?

**Comment by konung** [comment]
I'm guessing not. especially since Kendo now officially supports React: https://www.telerik.com/kendo-react-ui

**Comment by DanielRuf** [comment]
Huh? I thought they switched to Angular.

**Comment by DanielRuf** [comment]
Ah ok, thanks for the update.  Would be good to reference to this from the repo and archive it.

**Comment by konung** [comment]
Agreed. But I’m not one of the developers on this project :)  I was just watching the ticket, because was also interested in it. And then Kendo came out  with official support a couple of months ago with little fanfare, so I thought I’d mention it :D. I haven’t tried it myself yet.

Have a great day!

Nick Gorbikoff
nick.gorbikoff@gmail.com
